src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/custom_fixtures.yaml

```
@@ -17,6 +17,7 @@ sylius_fixtures:
                     options:
                         amount: 20
                         channel: 'FASHION_WEB'
+                        percentage_of_fulfilled_orders: 60
```

We need to define order fixtures twice. The current one should be changed to generate only 6, and new generation of orders with 14, to ensure 60% of orders.
```suggestion
                new_order:
                    options:
                        name: order
                        amount: 6
                        channel: 'FASHION_WEB'
                fulfilled_order:
                    options:
                        name: order
                        amount: 14
                        channel: 'FASHION_WEB'
                        fulfilled: true
```

---

I agree, this is the better approach 👌 (however 12/20 => 60% 😄)